src/mapdata.cpp

```diff
@@ -90,7 +90,8 @@ static const std::unordered_map<std::string, ter_bitflags> ter_bitflags_map = {
     { "CONNECT_TO_WALL",          TFLAG_CONNECT_TO_WALL }, // works with TFLAG_AUTO_WALL_SYMBOL
     { "CLIMBABLE",                TFLAG_CLIMBABLE },      // Can be climbed over
     { "GOES_DOWN",                TFLAG_GOES_DOWN },      // Allows non-flying creatures to move downwards
-    { "GOES_UP",                  TFLAG_GOES_UP }         // Allows non-flying creatures to move upwards
+    { "GOES_UP",                  TFLAG_GOES_UP },        // Allows non-flying creatures to move upwards
+    { "NO_FLOOR",                 TFLAG_NO_FLOOR }        // Things should fall when placed on this tile
```

Comment:
Protip (I learned that a while ago): the last entry in those initializer list can end with a comma. That makes it easier to add new entries as you don't have to change the previously last line.

For example see http://stackoverflow.com/questions/7043372/int-a-1-2-weird-comma-allowed-any-particular-reason